boards/arty_e21/src/main.rs

```diff
@@ -92,7 +93,7 @@ pub unsafe fn reset_handler() {
 
     let process_mgmt_cap = create_capability!(capabilities::ProcessManagementCapability);
     let main_loop_cap = create_capability!(capabilities::MainLoopCapability);
-    let memory_allocation_cap = create_capability!(capabilities::MemoryAllocationCapability);
+    let _memory_allocation_cap = create_capability!(capabilities::MemoryAllocationCapability);
```

Comment:
```suggestion
```

If this isn't needed any more, just delete it

Reply:
I was there, I deleted it now.